dev/com.ibm.ws.microprofile.metrics.2.0/resources/com/ibm/ws/microprofile/metrics/resources/Metrics.nlsprops

```diff
@@ -98,6 +98,9 @@ cpu.systemLoadAverage.description=Displays the system load average for the last
 #Cpu processCpuLoad message
 cpu.processCpuLoad.description=Displays the 'recent cpu usage' for the Java Virtual Machine process.
 
+#Cpu processCpuTime message
+cpu.processCpuTime.description=Displays the CPU time used by the process on which the Java Virtual Machine is running.
```

Comment:
Thanks- that does make it clearer. Can we rephrase as a complete sentence?

This message displays the CPU time that is used by the process on which the Java Virtual Machine is running.

Reply:
These are intentionally somewhat terse as they are just in a long long line of metrics that are output.  See the other descriptions in the file -- I think we need to be consistent with that style

Reply:
@dmuelle It seems best to leave the messages as is. Does that sound ok?

Reply:
yes i think the meaning comes across even without a complete sentence and this can stay as is for consistency